src/domain/user/models/user-service.ts

```diff
@@ -1,14 +1,14 @@
-import { User } from '../user';
+import { User, EditorAlias } from '../user';
 import { RestUserRepository } from '../infrastructure/rest-user';
 
 export class UserService {
-    userApiGetCurrentUser: RestUserRepository;
+    userApi: RestUserRepository;
     constructor(userAdapterUrl: string) {
-        this.userApiGetCurrentUser = new RestUserRepository(userAdapterUrl + '/current-user');
+        this.userApi = new RestUserRepository(userAdapterUrl);
     }
 
     async getCurrentUser(header: string): Promise<User> {
-        const user = await this.userApiGetCurrentUser.getCurrentUser(header);
+        const user = await this.userApi.getCurrentUser(header);
         switch (user.role) {
```

Comment:
There is a strong argument that this adaption between reviewer-role => continuum-role should be encapsulated in the RestUserRepository?